.github/workflows/update_remote_docs.yml

```diff
@@ -6,14 +6,21 @@ on:
       - "master"
     paths:
       - "docs/*"
+  pull_request:
+    branches:
+      - "master"
+    paths:
+      - "docs/*"
+  
```

Comment:
These lines can be removed. The update must be done only in the push on the master.